matter_server/server/server.py

```diff
@@ -123,7 +128,10 @@ def __init__(
         self.stack = MatterStack(self)
         # Initialize our (intermediate) device controller which keeps track
         # of Matter devices and their subscriptions.
-        self.device_controller = MatterDeviceController(self)
+        self.device_controller = MatterDeviceController(self, self.paa_root_cert_dir)
+        self.device_controller_api = MatterDeviceControllerAPI(
+            self, self.device_controller
```

Comment:
Now we're refactoring anyways, I would make these private and expose them publically using getters (properties)